scripts/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -f tmp/pids/server.pid ]; then
+  rm -f tmp/pids/server.pid
+fi
+
+bundle exec rake search:setup
+bundle exec rake db:migrate 2>/dev/null || bundle exec rake db:setup
```

Comment:
hooking Elasticsearch into these rake tasks so we should be able to just do `db:whatever` and that will handle postgres and ES.